.github/workflows/build-windows-executable-app.yaml

```
@@ -209,7 +210,7 @@ jobs:
         cp -r openms-package/share ../share
 
     - name: Set up Python (regular distribution)
-      uses: actions/setup-python@v5
+      uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)